mne/io/tests/test_meas_info.py

```diff
@@ -830,6 +830,15 @@ def test_repr_html():
     assert 'Projections' in info._repr_html_()
     info['projs'] = []
     assert 'Projections' not in info._repr_html_()
+    info['bads'] = []
+    assert 'None' in info._repr_html_()
+    info['bads'] = ['MEG 2443', 'EEG 053']
+    assert 'MEG 2443' in info._repr_html_()
+    assert 'EEG 053' in info._repr_html_()
+
+    html = info._repr_html_()
+    for ch in ['204 GRAD', '102 MAG', '9 STIM', '60 EEG', '1 EOG']:
+        assert ch in html
```

Comment:
To make sure things work for new sensor types, can you please also add a smoke test like `assert 'Good channels' in raw.info._repr_html_()` to [mne/io/tests/test_raw.py:_test_raw_reader](https://github.com/mne-tools/mne-python/blob/2adf9a63218e932399710618a095305ae0bd5d0d/mne/io/tests/test_raw.py#L255)? This monolithic function is used by all `mne/io/*` reading functions, so it's a good test that it works.